packages/tools/client-debugger/client-debugger/src/messaging/Messages.ts

```diff
@@ -8,10 +8,30 @@
  *
  * @public
  */
-export interface IDebuggerMessage<TData = unknown> {
+export interface IBaseDebuggerMessage<TData = unknown> {
```

Comment:
I wonder if a better naming distinction might be to name this "IDebuggerMessage", and the current `IDebuggerMessage` something like "ISourcedDebuggerMessage" - since the `source` label is the semantic distinction between the two.

Reply:
I did exactly that first (with exactly that name😆). I think I then switch tracks so I didn't have to update existing references to `IDebuggerMessage`. But I agree it makes more sense, I'll update for the final version of the PR.